README.md

```diff
@@ -3,11 +3,32 @@
 [![Join the chat at https://gitter.im/daostack/Lobby](https://badges.gitter.im/daostack/Lobby.svg)](https://gitter.im/daostack/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 # Arc
 
-Arc is the base layer of the DAO stack. It consists a set of smart contracts deployed on the Ethereum blockchain that define the basic building blocks and standard components that can be used to implement any DAO.
+Arc is the base layer of the DAO stack. It consists of a set of smart contracts deployed on the Ethereum blockchain that define the basic building blocks and standard components that can be used to implement any DAO.
+
+Arc is a modular, upgradeable platform that allows for a rapid natural selection of governance systems.
+
+![Arc Structure Diagram](https://user-images.githubusercontent.com/5414803/51004260-d7252180-1507-11e9-9be7-2018dbc7452d.jpeg)
+*<p align="center">Figure 1: Arc's structure</p>*
+
+**Every box in Figure 1 represents a smart contract** (to be precise, a proxy + logic contract).
```

Comment:
we still do not have proxies

Reply:
I see -- would it be best to just remove the parenthetical then?

Reply:
yes. remove it.

Reply:
Done.